### ISSUE TITLE
Issue with Collision Mechanics

### ISSUE THREAD

**Issue by EduardoZB** [jacobbriones1/Particles#1]
## Hey there Mr. Jacob!

I used your Python program as a template to start meddling with PyGame's capabilities! Thank you for your work!
Nevertheless, as I started to interact with it, I realized the **collision mechanics are not behaving as they should** (particles get thrown in the wrong direction, the hitting particle usually stays static after the collision).

After hitting the notebook and drawing a bunch of vectors and angles, I came up with a new *collide* function that improves the mechanic, although it doesn't take mass (size) into account... for now.

So, basically I'm asking if it's okay with you to open a new branch with this new function. Then you could check if it's up to your standards and if it could improve your program!
